ports/esp32/machine_sdcard.c

```diff
@@ -144,6 +144,7 @@ STATIC mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
         ARG_mosi,
         ARG_sck,
         ARG_cs,
+        ARG_bitrate,
```

Comment:
Hmm, is "bitrate" an accurate term here? IIUC multiple data lines can be hooked up and thus multiple bits can be transferred per clock cycle, thus "freq" or "clockrate" would be appropriate (esp-idf uses "freq").